jenkins/pipelines/cd/atom-jobs/build-common.groovy

```diff
@@ -838,33 +838,62 @@ cp audit/audit-1.so.md5 ${TARGET}/bin
 cp audit/audit-1.so ${TARGET}/bin
 """
 
+def usePod(){
+    return OS=="linux"
+}
+
+def upload_fileserver(local, remote){
+    if(usePod()){
+        container("ks3util"){
```

Comment:
Please do not hardcode the container name, and it is better be judged in upper level.

Reply:
hard to determine its upper level